spec/support/feature_helper.rb

```diff
@@ -14,5 +14,17 @@ def sign_in(user)
     end
     expect( page ).to have_content("Signed in successfully.")
   end
+
+  def authenticated_user(options={})
+    user_options = {
+      email: "email#{rand}@fake.com",
+      password: 'password'
+    }.merge(options)
+
+    user = User.new(user_options)
+    user.skip_confirmation!
+    user.save
+    user
+  end
```

Comment:
`authenticated_user(options={})` is good. Thought you might appreciate to see how this could also be done using  `FactoryGirl`: https://github.com/eliotsykes/rspec-rails-examples/blob/master/spec/factories/users.rb . 

Example uses of resulting user factory:
- https://github.com/eliotsykes/rspec-rails-examples/blob/332cd288a202f5ecd7a383c0a2cadf04a45abca7/spec/features/user_login_and_logout_spec.rb#L30
- https://github.com/eliotsykes/rspec-rails-examples/blob/332cd288a202f5ecd7a383c0a2cadf04a45abca7/spec/features/user_login_and_logout_spec.rb#L7